### PR TITLE
Add Twin Trials Contest Results news post

### PR DESCRIPTION
**Pull request by pishifat**
shh

### CODE REVIEW

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 1]
```diff
@@ -177,3 +179,101 @@ The storyboarding phase prizes will be announced after the beatmapping phase end
...
+
+The following categories were evaluated across all difficulties in a submission:
+
+- **Musical representation** (20 points max) - How the mapset expresses the song through overall structure, patterning, and hitsounding (if relevant to the game mode)
+- **Creativity** (20 points max) - How innovative the mapset is relative to other submissions and the mode's general mapping practices
+- **Judges impression** (10 points max) - General impression (and anything else of note in a submission that doesn't fit a preset categories)
+
+### osu!
+
+| Rank | Team | Map | Representation | Creativity | Impression | X | I | H | N | E | Total |
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:201]: why not go for something sane like E N H I X order?

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 2]
```diff
@@ -177,3 +179,101 @@ The storyboarding phase prizes will be announced after the beatmapping phase end
...
+| :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- |
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:202]:
```suggestion
| :-: | :-- | :-: | :-- | --: | --: | --: | --: | --: | --: | --: | --: |
```

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 3]
```diff
@@ -177,3 +179,101 @@ The storyboarding phase prizes will be announced after the beatmapping phase end
 - **Submissions must not be Qualified until after the community vote ends.**
 - **Each team cannot create more than one submission per game mode.**
 - **A user cannot participate in more than one team per game mode.**
+
+## Results
+
+Each of the 5 difficulties had its own judging category, aiming to evaluate the difficulty's overall quality. These were weighted according to each difficulty's complexity:
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:185]:
```suggestion
Each of the 5 difficulties had its own judging category, aiming to evaluate its overall quality. The point distribution was weighted according to each difficulty's complexity:
```

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 4]
```diff
@@ -177,3 +179,101 @@ The storyboarding phase prizes will be announced after the beatmapping phase end
...
+
+- **Easy** (8 points max)
+- **Normal** (9 points max)
+- **Hard** (10 points max)
+- **Insane** (11 points max)
+- **Expert** (12 points max)
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:191]:
```suggestion
| Difficulty | Max points |
| :-- | :-- |
| Easy | 8 |
| Normal | 9 |
| Hard | 10 |
| Insane | 11 |
| Expert | 12 |
```

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 5]
```diff
@@ -177,3 +179,101 @@ The storyboarding phase prizes will be announced after the beatmapping phase end
...
+
+The following categories were evaluated across all difficulties in a submission:
+
+- **Musical representation** (20 points max) - How the mapset expresses the song through overall structure, patterning, and hitsounding (if relevant to the game mode)
+- **Creativity** (20 points max) - How innovative the mapset is relative to other submissions and the mode's general mapping practices
+- **Judges impression** (10 points max) - General impression (and anything else of note in a submission that doesn't fit a preset categories)
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:197]:
```suggestion
| Judgement category | Max points | Description |
| :-- | :-- | :-- |
| Musical representation | 20 | How the beatmap expresses the song through overall structure, patterning, and hitsounding (if relevant to the game mode) |
| Creativity | 20 | How innovative the beatmap is relative to other submissions and the game mode's general mapping practices |
| Judges' impression | 10 | General impression and anything else of note in a submission that does not fit existing categories |
```

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 6]
```diff
@@ -177,3 +179,101 @@ The storyboarding phase prizes will be announced after the beatmapping phase end
...
+| 9 | [Salute, bebra](https://mappersguild.com/contests/results?submission=64254979ba751857a432b4ad) | [#1](https://osu.ppy.sh/beatmapsets/1973571) | 114 | 127 | 51 | 75 | 71 | 58 | 54 | 52 | **602** |
+| 10 | [Nightcord at Various Times](https://mappersguild.com/contests/results?submission=64123a2b146d25b6715501cf) | [#1](https://osu.ppy.sh/beatmapsets/1972105) | 108 | 110 | 44 | 61 | 58 | 55 | 41 | 46 | **523** |
+| 11 | [FUMO 20 GRAMMI IN 10 SEC CAR](https://mappersguild.com/contests/results?submission=64123a2c146d25b6715501df) | [#1](https://osu.ppy.sh/beatmapsets/1972425) | 110 | 89 | 40 | 62 | 58 | 55 | 50 | 49 | **513** |
+| 12 | [nilou mains](https://mappersguild.com/contests/results?submission=64123a2b146d25b6715501d3) | [#1](https://osu.ppy.sh/beatmapsets/1972507) | 105 | 104 | 43 | 62 | 59 | 52 | 46 | 36 | **507** |
+| 13 | [Double Trouble](https://mappersguild.com/contests/results?submission=64123a2b146d25b6715501c7) | [#1](https://osu.ppy.sh/beatmapsets/1972412) | 59 | 55 | 19 | 32 | 27 | 28 | 21 | 24 | **265** |
+
+### osu!taiko
+
+| Rank | Team | Map | Musical rep. | Creativity | Impression | X | I | H | N | E | Total |
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:219]: also these headers should be consistent for all of these tables

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2023/2023-05-15-results-twin-trials-beatmapping-phase.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+#### Judging winner
+
+[![PepeBusinessCorp team banner](/wiki/shared/news/2023-04-17-twin-trials-beatmap-voting-open/osu/PepeBusinessCorp.png)](https://osu.ppy.sh/beatmapsets/1971987)
+
+With so many veteran mappers entering **Twin Trials**, everyone knew the competition would be fierce.
+
+On the judges' side, [DeviousPanda](https://osu.ppy.sh/users/4966334) and [iljaaz](https://osu.ppy.sh/users/8501291) proved they have what it takes to create 5 consistently awesome difficulties. [Their submission](https://osu.ppy.sh/beatmapsets/1971987) explored the more challenging side of what the song had to offer, while maintaining a crispy clean style throughout.
+
+With this outcome, the two mappers have earned the profile badge below, [2 points of progress towards the *Elite Mapper* user title](/wiki/Contests/Contest_points), and their map will be used as the base for the storyboarding phase of **Twin Trials** in the soon future!
```
Comment (Walavouchey) [inline news/2023/2023-05-15-results-twin-trials-beatmapping-phase.md:63]: ???

```suggestion
With this outcome, the two mappers have earned the profile badge below, [2 points of progress towards the *Elite Mapper* user title](/wiki/Contests/Contest_points), and their map will be used as the base for the storyboarding phase of **Twin Trials** in the near future!
```

File: news/2023/2023-05-15-results-twin-trials-beatmapping-phase.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+layout: post
+title: "Results - Twin Trials: Beatmapping phase"
+date: 2023-05-15 19:30:00 +0000
+---
+
+After months of mapping, judging, and voting, the first half of Twin Trials is coming to a close! It's time to see which duos take home the gold for every game mode.
+
+![](https://assets.ppy.sh/contests/166/header.jpg)
+
+[3 months ago](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase), we launched [**Twin Trials**](/wiki/Contests/Twin_Trials_Contest), a contest with two phases: *beatmapping* and *storyboarding*, all centered around some new [osu! originals](/wiki/osu!_originals) created by one of our old-school [Featured Artists](https://osu.ppy.sh/beatmaps/artists) [*James Landino*](https://osu.ppy.sh/beatmaps/artists/39).
```
Comment (Walavouchey) [inline news/2023/2023-05-15-results-twin-trials-beatmapping-phase.md:11]:
```suggestion
[3 months ago](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase), we launched [**Twin Trials**](/wiki/Contests/Twin_Trials_Contest), a contest with two phases: *beatmapping* and *storyboarding*, all centred around some new [osu! originals](/wiki/osu!_originals) created by one of our old-school [Featured Artists](https://osu.ppy.sh/beatmaps/artists) [*James Landino*](https://osu.ppy.sh/beatmaps/artists/39).
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 9]
```diff
@@ -177,3 +179,104 @@ The storyboarding phase prizes will be announced after the beatmapping phase end
...
+- **Insane playlist**
+  - osu!: [fieryrage](https://osu.ppy.sh/users/3533958)
+  - osu!taiko: [Yukiama95](https://osu.ppy.sh/users/18535502)
+  - osu!catch: [AnApple7](https://osu.ppy.sh/users/12567935)
+  - osu!mania: [mashu](https://osu.ppy.sh/users/12028216)
+- **Expert playlist**
+  - osu!: [fieryrage](https://osu.ppy.sh/users/3533958)
+  - osu!taiko: [Grape_Tea](https://osu.ppy.sh/users/9540073)
+  - osu!catch: [Kimitakari](https://osu.ppy.sh/users/4741164)
+  - osu!mania: [mashu](https://osu.ppy.sh/users/12028216)
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:282]:
```suggestion
|  | ![](/wiki/shared/mode/osu.png) osu! | ![](/wiki/shared/mode/taiko.png) osu!taiko | ![](/wiki/shared/mode/catch.png) osu!catch | ![](/wiki/shared/mode/mania.png) osu!mania |
| :-- | :-- | :-- | :-- | :-- |
| **Easy playlists** | ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958) | ::{ flag=JP }:: [Yukiama95](https://osu.ppy.sh/users/18535502) | ::{ flag=VN }:: [\[ Primakien \]](https://osu.ppy.sh/users/23941998) | ::{ flag=US }:: [mashu](https://osu.ppy.sh/users/12028216) |
| **Normal playlists** | ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958) | ::{ flag=JP }:: [Yukiama95](https://osu.ppy.sh/users/18535502) | ::{ flag=LV }:: [AnApple7](https://osu.ppy.sh/users/12567935) | ::{ flag=US }:: [mashu](https://osu.ppy.sh/users/12028216) |
| **Hard playlists** | ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958) | ::{ flag=JP }:: [Yukiama95](https://osu.ppy.sh/users/18535502) | ::{ flag=TR }:: [Yasinnnnnn](https://osu.ppy.sh/users/17482520) | ::{ flag=US }:: [mashu](https://osu.ppy.sh/users/12028216) |
| **Insane playlists** | ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958) | ::{ flag=JP }:: [Yukiama95](https://osu.ppy.sh/users/18535502) | ::{ flag=LV }:: [AnApple7](https://osu.ppy.sh/users/12567935) | ::{ flag=US }:: [mashu](https://osu.ppy.sh/users/12028216) |
| **Expert playlists** | ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958) | ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) | ::{ flag=RU }:: [Kimitakari](https://osu.ppy.sh/users/4741164) | ::{ flag=US }:: [mashu](https://osu.ppy.sh/users/12028216) |
```

File: wiki/Contests/Twin_Trials_Contest/en.md [hunk 10]
```diff
@@ -20,9 +20,9 @@ The storyboarding phase will be revealed after the beatmapping phase ends.
 | Event | Timestamp |
 | --: | :-- |
 | Announcement | 2023-02-01 |
-| Submission | 2023-02-01 to 2023-03-15 |
-| Judging | 2023-03-15 to 2023-04-05 |
-| Community vote and lazer playlists | 2023-04-10 to 2023-04-24 |
+| Submission | 2023-02-01/2023-03-15 |
+| Judging | 2023-03-15/2023-04-05 |
+| Community vote and lazer playlists | 2023-04-10/2023-04-24 |
 | Results | 2023-05-01 |
```
Comment (Walavouchey) [inline wiki/Contests/Twin_Trials_Contest/en.md:26]: date?

**Review by Walavouchey — APPROVED** [review]

File: news/2023/2023-05-15-results-twin-trials-beatmapping-phase.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: "Results - Twin Trials: Beatmapping phase"
+date: 2023-05-15 19:30:00 +0000
```
Comment (Walavouchey) [inline news/2023/2023-05-15-results-twin-trials-beatmapping-phase.md:4]:
```suggestion
date: 2023-05-15 21:00:00 +0000
```

**Review by Walavouchey — APPROVED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**